src/main.rs

```diff
@@ -120,11 +40,20 @@ fn main() -> Result<()> {
             }
         };
```

Comment:
```rust
    let generated = gen::generate(
        &contents,
        gen::ParserOpts {
            mode: gen::ParserMode::Autoload {
                // TODO: Calculate this correctly...
                prefix: "ccomplete".to_string(),
            },
        },
    )
    .unwrap_or_else(|err| {
        println!("  FAILED: {}", err.1);
        err.0
    });
```
This whole `generated` thing might look slightly clearer with `unwrap_or_else()`.